v1_0/tasks.pbtxt

```diff
@@ -0,0 +1,205 @@
+task {
+  name: "Image classification"
+  dataset {
+    name: "Imagenet classification validation set"
+    type: IMAGENET
+    path: "/sdcard/mlperf_datasets/imagenet/img"
+    groundtruth_src: "@assets/imagenet_val_full.txt"
```

Comment:
Here and below:
`groundtruth_src looks` not good... I think it's a bad way to put links to internal resources in a downloadable file. By the way, it link makes sense for the android application only. In the case of iOS, this link will be not valid as I understand.